src/peerbook/libp2p_peerbook.erl

```diff
@@ -185,7 +185,7 @@ random(Peerbook=#peerbook{store=Store, stale_time=StaleTime}, Exclude, Pred, Tri
                 false ->
```

Comment:
map rev over the exclude list or it won't match right, I think

Reply:
good catch!

Reply:
done